godtools/App/Features/Onboarding/Views/OnboardingQuickStartCell.swift

```diff
@@ -13,8 +13,10 @@ class OnboardingQuickStartCell: UITableViewCell {
     static let nibName: String = "OnboardingQuickStartCell"
     static let reuseIdentifier: String = "OnboardingQuickStartCellReuseIdentifier"
     
+    @IBOutlet weak private var contentBackgroundView: UIView!
     @IBOutlet weak private var titleLabel: UILabel!
-    @IBOutlet weak private var subtitleLabel: UILabel!
+    @IBOutlet weak private var actionLabel: UILabel!
+    @IBOutlet weak private var actionIcon: UIImageView!
```

Comment:
Do we need this outlet, if we're not using it in this view class?

Reply:
Hey @reldredge71 you're right that is not being used and we really don't have to connect it, it's not necessary or required.  It's more of a personal preference on my part, it helps me quickly see what elements make up a view without opening the .xib.  In the future migrating to SwiftUI will be nice because then there won't be anymore .xib and swift file.  All UI will be in one place.